Defs/Ammo/Rifle/5x50mmCaseless.xml

```diff
@@ -0,0 +1,187 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should probably go into 'Advanced' rather than 'Rifle', despite it 5x100mm previously having been there, since most of our existing rifle rounds are real cartridges.